### ISSUE TITLE
Parking spots 192-203 - 109-124 and 1-4 are all EOR arming areas - not really parking spots

### ISSUE THREAD

**Issue by hbjonsson** [132nd-vWing/ATRM#119]
As per the title - several jets have their parking spawn in these locations. While that is handy for quick launches, it might be rather rather detrimental to immersion and wing cohesion during LFEs.

I would suggest discontinuing the use of these spots for the reasons above, as well as the fact that these EOR spots might be in use by aircraft taxiing out to the runway(s) if conducting EOR armament / hot pit refueling procedures as per RL.

I have devised an updated .miz where parking locations have been moved ( and aircraft re-labeled accordingly ) along with an updated speed for AR201 for testing purposes. 

[ATRM_2.5.6.395_Jester.miz.zip](https://github.com/132nd-vWing/ATRM/files/6085133/ATRM_2.5.6.395_Jester.miz.zip)

**Comment by 132ndNeck** [comment]
As mentioned in Discord, the reason for having some jets availeble at either end of the runway is to make it possible to shorten down the transit time in taxi when needed. Another reason why jets have been spread out, is that at times, people can experience FPS issues, and these get worse if there are a lot of jets in the close proximity.
So, yes immersion and wing cohesion is important, but we also need to take real life into consideration and technical status of DCS and each members computer.

But, I am happy with a move of the startup spots out of the EOR areas.
But some aircrafts need to availeble at each side of the runway, as an option for pilots if needed.

So, to sum it up, happy with creating the parking in the center as suggested by Jester, but need some more jets availeble at either end as a option.
